dataprofiler/profilers/profiler_options.py

```diff
@@ -129,6 +129,9 @@ def _validate_helper(self, variable_path='BooleanOption'):
         :return: list of errors (if raise_error is false)
         :rtype: list(str)
         """
+        if not isinstance(variable_path, str):
```

Comment:
See comment below on repetition in BaseColumnOption. I could add the check to the BaseOption class instead (raise a ValueError before the NotImplementedError) to avoid any repetition.